watch.go

```diff
@@ -121,6 +121,18 @@ func (ss *ServerSet) Watch() (*Watch, error) {
 func (w *Watch) Endpoints() []string {
 	w.lock.RLock()
 	defer w.lock.RUnlock()
+	endpoints := make([]string, 0, len(w.endpoints))
+	for _, e := range w.endpoints {
+		endpoints = append(endpoints, net.JoinHostPort(e.ServiceEndpoint.Host, strconv.Itoa(e.ServiceEndpoint.Port)))
+	}
+
+	sort.Strings(endpoints)
+	return endpoints
+}
+
+func (w *Watch) EndpointEntities() []Entity {
```

Comment:
Looks good! thank you! Maybe add a comment here and then rebase off the current master so I can merge.